djl-serving/multi-node/lws.yaml

```diff
@@ -18,8 +18,8 @@ spec:
             volumeMounts:
               - name: dshm
                 mountPath: /dev/shm
-              - name: model-storage
-                mountPath: /opt/ml/model
+              - name: cache-storage
```

Comment:
I think in real world, user still mount to /opt/ml/model, right?

Reply:
Added back.